File: AtCoder/ABC/B/269/B.py
start, goal = [], []
for i in range(10):
    s = input()
    for j in range(10):
        if s[j] == '#':
            if not start: start = [i+1, j+1]
            goal = [i+1, j+1]
print(*[start[0], goal[0]], sep=' ')
print(*[start[1], goal[1]], sep=' ')

File: AtCoder/ABC/B/269/B.rb
start = Array.new
goal = Array.new
10.times do |i|
    s = gets.chomp
    s.chars.each_with_index do |c, j|
        if c == '#'
            start = [i+1, j+1] if start.empty?
            goal = [i+1, j+1] 
        end
    end
end
puts "#{start[0]} #{goal[0]}\n#{start[-1]} #{goal[-1]}"